### ISSUE TITLE
`collect()` needs to receive results as a list

### ISSUE THREAD

**Issue by diazrenata** [weecology/MATSS#67]
If I run the pipeline with `results_lda` generated according to this: 
```
  results = target(collect(analysis, ignore(analyses)),
                     transform = combine(analysis, .by = fun))
```
[...]
I get this error:
```
> ## Run the pipeline
> make(pipeline, cache = cache)
target results_lda
fail results_lda
Error: Target `results_lda` failed. Call `diagnose(results_lda)` for details. Error message:
  unused arguments (analysis_lda_jornada_data, analysis_lda_sgs_data, analysis_lda_bbs_data, analysis_lda_sdl_data, analysis_lda_mtquad_data, analysis_lda_bad_portal, ignore(analyses))
```
which I think is happening because `collect` is getting all the `analysis` names as individual arguments instead of as a list. Here is how `collect` is defined:
```
## Define how results are collected
collect <- function(list_of_results, plan)
{
    names(list_of_results) <- all.vars(match.call()$list_of_results)
    list_of_results
}
```

Changing the `results` step of the pipeline fixes it:

```
 results = target(collect(list(analysis), ignore(analyses)),
                     transform = combine(analysis, .by = fun))
```

See [commit in my branch](https://github.com/weecology/MATSS-pipeline/commit/64ea083bdaebe9dd9fbb4b86a31c4f93adaeabfc).

**Comment by ha0ye** [comment]
Huh. Weird that this issue is occurring. I recall not having to explicitly indicate that a list was to be created, but maybe I should have?

**Comment by ha0ye** [comment]
Tracked down to change in Drake DSL syntax from 6.2.1.9002 to 6.2.1.9003. Resolved by PR #68
